### PR TITLE
Fix dao md: "The Blockchain Congress" chapter.

### PR DESCRIPTION
**Pull request by sot528**
I fixed "The Blockchain Congress" chapter of **dao.md**.

- I changed explanations and code according to the latest Ethereum Wallet(0.8.10).

### CODE REVIEW

File: views/content/dao.md [hunk 1]
```diff
@@ -128,7 +128,6 @@ The way this particular democracy works is that it has an **Owner** which works
                members[id] = Member({member: targetMember, memberSince: now, name: memberName});
             } else {
                 id = memberId[targetMember];
-                Member m = members[id];
```
Comment (alexvandesande) [inline views/content/dao.md]: Do not remove this line. Instead I believe line 128 should be moved to 132. Need to test

Comment (sot528) [inline views/content/dao.md]: @alexvandesande 
Thank you for your reply sande.
I tried you said, however still Ethereum wallet said that:


        Could not compile source code. 
        
        Unused local variable
            Member m = members[id];
            ^------^

And to me, it looks unnecessary.
Maybe my understanding of code above is not correct?
Or did I mistake something?  
  

Ethereum Wallet v0.8.10

Comment (alexvandesande) [inline views/content/dao.md]: What I believe the code should be is:

```
if (memberId[targetMember] == 0) {		             
    memberId[targetMember] = members.length;		              
    id = members.length++;
} else {		             
    id = memberId[targetMember];		
}

members[id] = Member({member: targetMember, memberSince: now, name: memberName});
````

So you're right, that line must be removed and then the members[id] line must be changed..

File: views/content/dao.md [hunk 2]
```diff
@@ -930,14 +929,13 @@ Once the minimum amount of time has passed, anyone can execute the transaction [
...
-               id = members.length++;
-               members[id] = Member({member: targetMember, memberSince: now, name: memberName});
+                memberId[targetMember] = members.length;
+                id = members.length++;
             } else {
                 id = memberId[targetMember];
-                Member m = members[id];
             }
 
+            members[id] = Member({member: targetMember, memberSince: now, name: memberName});
```
Comment (sot528) [inline views/content/dao.md]: Sorry here is other chapter.
But it had same problem as titled chapter.

**Pull request merged**